driver-core/src/main/com/mongodb/ConnectionString.java

```diff
@@ -1127,14 +1127,16 @@ public WriteConcern getWriteConcern() {
     /**
      * Returns true if writes should be retried if they fail due to a network error, and false otherwise
      *
-     * @return the retryWrites value, or false if unset
+     * <p>Starting with the 3.11.0 release, the default value is true</p>
+     *
+     * @return the retryWrites value, or true if unset
      * @since 3.6
      * @mongodb.server.release 3.6
      * @deprecated Prefer {@link #getRetryWritesValue()}
      */
     @Deprecated
     public boolean getRetryWrites() {
-        return retryWrites == null ? false : retryWrites;
```

Comment:
Why not use boolean for retryWrites and default it to true?

Reply:
Then getRetryWritesValue won't work properly.  We need to know whether the value was set in the connection string in order to properly implement the MongoClientSettings#applyConnectionString

Reply:
Ack